### ISSUE TITLE
Option to switch to clearnet in webui to speed up initial blockchian sync

### ISSUE THREAD

**Issue by rootzoll** [raspiblitz/raspiblitz#3500]
like assumed in #3329 ... often initial sync just behind Tor is slow. Give option to switch (just bitcoind) to additionally clearnet to speed up sync

**Comment by rootzoll** [comment]
Maybe this is not needed when we support assumeUTXO .. push further in the future.

**Comment by rootzoll** [comment]
Pushing this to v1.12.0 for same reasons like AssumeUTXO - see https://github.com/raspiblitz/raspiblitz/issues/1178#issuecomment-2381257789

It still makes sense to combine this with AssumeUTXO because we had setups in workshops where Tor is blocked or slowed down extremly .. so any option that speeds up blockchain resync is needed for v1.12.0 to reduce the migration pain.

**Comment by scarolan** [comment]
Yes please! I'm going on ten days now with a Raspberry Pi 4 still at only 20%.... 😔

Giving the user an option to do their initial sync over clearnet would be a big improvement.

I disabled tor and sync immediately started working again; it advanced 1% in about ten minutes.
